pkg/imgpkg/imagedesc/image_ref_descriptors.go

```diff
@@ -28,8 +28,9 @@ type Registry interface {
 }
 
 type Metadata struct {
-	Ref regname.Reference
-	Tag string
+	Ref    regname.Reference
+	Tag    string
+	Labels map[string]interface{}
```

Comment:
i think it might be worth making this map[string]string (follows more conventional labels/annotations signature within kubernetes ecosystem)

Reply:
ok done

Reply:
I think this is a breaking change...

if someone were to use a tarball generated with v0.14.0, the tarball has `nil` as the map value.

I think they will get a `panic` complaining that they are unable to convert to a string.

The workaround would be to re-generate the tarball with the newer version of imgpkg.